test/repl/proxysql_repl_tests/exec_shun_test.sh

```
@@ -196,18 +190,11 @@ fn_start () {
 
   # Start ProxySQL
   SECONDS=0
-  echo "[`date '+%Y-%m-%d %H:%M:%S'`] >>> Starting ProxySQL ..."
-  echo "[`date '+%Y-%m-%d %H:%M:%S'`] >>> ProxySQL binary '$WORKSPACE/src/proxysql'"
-  echo "[`date '+%Y-%m-%d %H:%M:%S'`] >>> ProxySQL config '$REPL_TESTS_PATH/conf/proxysql/proxysql.cnf'"
-  echo "[`date '+%Y-%m-%d %H:%M:%S'`] >>> ProxySQL datadir '$REPL_TESTS_PATH/conf/proxysql'"
-  echo "[`date '+%Y-%m-%d %H:%M:%S'`] >>> ProxySQL logging '$REPL_INFRA_DATADIR/proxysql/proxysql.log'"
+  echo "[`date '+%Y-%m-%d %H:%M:%S'`] >>> Starting ProxySQL..."
   cd "$WORKSPACE/src"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)